frontend/src/Pages/SignIn.tsx

```diff
@@ -79,6 +82,10 @@ const SignIn = () => {
             <Link to="/signup" className="ui fluid large basic button">
               Create an account
             </Link>
+            <p className="ui fluid large centered">or</p>
+            <Link to="/admin" className="ui fluid large button">
+              Are you an administrator?
+            </Link>
```

Comment:
Hmm its not ideal to create a route for user to access the admin pages. better for admin to just access the login page from the url itself. So this can be removed.

Reply:
Ok sir, will remove right away.